src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/admin/moderation/ModerationPageSpec.groovy

```diff
@@ -0,0 +1,67 @@
+package pl.tomaszdziurko.jvm_bloggers.view.admin.moderation
+
+import org.apache.wicket.protocol.http.WebApplication
+import org.apache.wicket.util.tester.FormTester
+import org.apache.wicket.util.tester.WicketTester
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.util.ReflectionTestUtils
+import pl.tomaszdziurko.jvm_bloggers.SpringContextAwareSpecification
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.DummyBlogPostsFetcher
+import pl.tomaszdziurko.jvm_bloggers.blogs.DummyBloggersDataFetcher
+import pl.tomaszdziurko.jvm_bloggers.view.admin.AdminDashboardPage
+import pl.tomaszdziurko.jvm_bloggers.view.login.LoginPage
+
+class ModerationPageSpec extends SpringContextAwareSpecification {
+
+    private WicketTester tester
+
+    @Autowired
+    WebApplication wicketApplication
+
+    @Autowired
+    DummyBloggersDataFetcher dummyBloggersDataFetcher;
```

Comment:
I refactored our tests to be able to do it easily in #182. You will get this feature without additional Spring configuration.